src/material/form-field/form-field.ts

```diff
@@ -409,25 +418,31 @@ export class MatFormField
   }
 
   /** Initializes the registered form field control. */
-  private _initializeControl() {
+  private _initializeControl(previousControl: MatFormFieldControl<unknown> | null) {
```

Comment:
why pass previousControl in, can't we just read it off the class like we do for control?

Reply:
I wanted to keep the code that re-assigns it in the `ngAfterContentChecked` since that's the only place where we check it.